app/Http/Controllers/Profile/ProfileController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers\Profile;
+
+use Kris\LaravelFormBuilder\FormBuilderTrait;
+use Illuminate\Http\Request;
+use App\Member;
+use App\Forms\UpdateProfileForm;
+use App\Http\Requests;
+use App\Http\Controllers\Controller;
+use Auth;
+use JsValidator;
+use Session;
+
+class ProfileController extends Controller
+{
+    use FormBuilderTrait;
+	
+    public function show()
+    {
+		$user = Auth::user();
```

Comment:
Yup that's right! Oh, hmm - maybe consider using a different editor? (You're on Linux since you're using wine, right?) I am bias and recommend Sublime (or vim) but there are plenty of other nice editors and you won't have to go through wine...